test/Manager.t.sol

```diff
@@ -342,7 +356,7 @@ contract TestManager is Test, IManagerEvents {
             0,
             abi.encodePacked(address(manager)),
             abi.encodePacked(address(0)),
-            abi.encode(EndpointStructs.EndpointMessage("hello", "world", "payload"))
+            abi.encode(EndpointStructs.EndpointMessage(0x9945FF10, "payload"))
```

Comment:
can you please update the tests to wrap the `ManagerMessage` in the `EndpointMessage` and not the other way around?
You'll need to update the `DummyEndpoint.receiveMessage` logic.

Reply:
will update this in a separate PR along with other test refactors. currently tests are not setup to test the wrapping logic because it skips the `deliverToManager` path

Reply:
Sounds good and thanks for creating https://github.com/wormhole-foundation/example-native-token-transfers/issues/48